site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    gemini_ReAct_chat = ReAct(model='gemini-pro', ReAct_prompt='/content/model_instructions.txt')
</pre><p> </p><p>Uber nit - just use <code>model_instructions.txt</code> as the filename so that the notebook works outside of Colab. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>